Flashcards.Dejmenek/Flashcards.Dejmenek/Controllers/StudySessionsController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Flashcards.Dejmenek.DataAccess.Repositories;
+using Flashcards.Dejmenek.Helpers;
+using Flashcards.Dejmenek.Models;
+using Flashcards.Dejmenek.Services;
+using Spectre.Console;
+
+namespace Flashcards.Dejmenek.Controllers
+{
+    public class StudySessionsController
+    {
+        private readonly StudySessionsRepository _studySessionsRepository;
+        private readonly UserInteractionService _userInteractionService;
+        public int Score { get; private set; }
```

Comment:
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Flashcards/assets/57032138/c37c322f-256a-447f-9b49-76b5ad8b5ba9)
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Flashcards/assets/57032138/a144b1d7-50c9-4778-84af-c4364aa68c38)
The global score property is not reset at the beginning of a study session as such, previous scores are added to the current session's score. Consider resetting your score counter before starting a session 😄 

This bug also affects this report
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Flashcards/assets/57032138/be1d9834-92cc-4f00-a346-77bd2379def4)
I studied the polish just once there's no way I got 7 points 😆